src/incogniaApi.ts

```diff
@@ -101,12 +101,32 @@ export class IncogniaApi {
   }
 
   async registerLogin(props: RegisterLoginProps): Promise<TransactionResponse> {
+    const { installationId, accountId } = props || {}
+    if (!installationId || !accountId) {
+      throw new IncogniaError('No installationId or accountId provided')
+    }
```

Comment:
This kind of validation is already made by the API, do we want to repeat validations?

Without them would be easier to evolve the wrapper that would be only a middle resource to make the request.

Reply:
I saw [here](https://github.com/inloco/incognia-node/pull/64/commits/c081be6ff8d2ce3081c70398b7ae103a0bd42c71#diff-b69421f88ed56684b73ccaa0323a718018877e60db85ee95ed0b2ebaa768b6f7L139) that this is the current behavior

Reply:
Good question...

Today we have two types of errors... `IncogniaError` (from the wrapper) and `IncogniaApiError` from the API.

If we remove these wrapper validations, it would be simpler... 

However, I don't know if there is any collateral effect. 

🤔

Reply:
I imaged so :doggo-sad:

Reply:
Anyway, this could be removed in another version (probably with a breaking change).